.github/ISSUE_TEMPLATE/meeting_setup.md

```diff
@@ -40,3 +40,13 @@ to maximise the notification period and increase attendance but reminders can be
       Slack](https://nhsrcommunity.slack.com/).
 + [ ] **Sheffield Digital Slack** : If you are a member you can announce the event on the [Sheffield Digital Slack](https://sheffield.digital/slack).
 + [ ] **Royal Statistical Society Sheffield Local Group** Contact the [RSS Sheffield Local](https://rss.org.uk/membership/rss-groups-and-committees/groups/sheffield/) group and ask them to disseminate details of the meeting (contacts are currently [Esther Herbert](mailto:e.herbert@sheffield.ac.uk) but may change over time).
+
+
+## Post Event
+
++ [ ] **Add material**: If the speakers had slides, ask them for the slides and add them under "SheffieldR.github.io/posts
```

Comment:
Just to clarify for future users we could say...

```suggestion
+ [ ] **Add material to web-page**: Add a link to slides to the post under `SheffieldR.github.io/posts/yyyy-mm-dd/index.qmd`. If the speaker hosted the slides on GitHub pages or Google Slides this can just be a standard link to the URL (although it may be desirable to make a PDF of Google Slides and include it on the site). If the speakers had slides as a PDF of PowerPoint slides request a copy and place them in the same directory as the web-page adding a link to `index.qmd`. 
```